openslides_backend/action/actions/user/update.py

```diff
@@ -140,8 +140,11 @@ def update_instance(self, instance: dict[str, Any]) -> dict[str, Any]:
                 raise PermissionException(
                     "A superadmin is not allowed to set himself inactive."
                 )
-        if instance.get("is_active") and not user.get("is_active"):
-            self.check_limit_of_user(1)
+        if instance.get("is_active"):
+            if not user.get("is_active"):
+                self.check_limit_of_user(1)
+        elif user.get("is_active"):
+            self.auth.clear_sessions_by_user_id(instance["id"])
```

Comment:
"If is_action is not in payload (instance) and user is_active, then clear session", this doesn't seem right.
The is_active payload is **optional** as far as I know.

Reply:
That should be repaired now.